backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -17,6 +17,9 @@ const { NotImplemented } = require('http-errors')
 
 const { isHttpError } = require('@gardener-dashboard/request')
 const { dashboardClient } = require('@gardener-dashboard/kube-client')
+const {
+  getManagedSeedOrUndefined
```

Comment:
I think it is a normal behavior that a function does not return something or returns null. We should not put input parameter names or possible result values into the name of the function. The name of a function should only describe what the function does. If it does not find a managed seed it cannot return one.

Reply:
✅